swisstopo-boreholeML/swisstopoPostgreSql_to_BoreholeML30.halex.alignment.xml

```diff
@@ -740,6 +740,16 @@ if (location_x_bho &amp;&amp; location_y_bho &amp;&amp; elevation_z_bho &amp;&am
         </target>
         <parameter value="https://www.infogeo.de/boreholeml/3.0/codelists/DrillingMethodList.xml" name="value"/>
     </cell>
+    <cell relation="eu.esdihumboldt.hale.align.assign" id="C93318591-5e39-49b4-be0e-6ab5928bf75e" priority="normal">
+        <target>
+            <property>
+                <type name="BoreholeType" ns="http://www.infogeo.de/boreholeml/3.0"/>
+                <child name="drillingProcess" ns="http://www.infogeo.de/boreholeml/3.0"/>
+                <child name="nilReason"/>
+            </property>
+        </target>
+        <parameter value="unknown" name="value"/>
```

Comment:
Why not unpopulated as it is not provided for all objects?

Reply:
Can I use 'unpopulated' for non-INSPIRE datasets? 
The schema in hale studio says that this is not one of the allowed values (?) 
![grafik](https://github.com/wetransform-os/swissgeol-geosciml-mapping/assets/116007248/7dd510b6-71f2-484b-af67-7e5aea3f648a)

Reply:
Ah no you are right, of couse. I did not think about the fact that this is not an INSPIRE dataset. There might be an option if the list is extended, but that does definitely not make sense here IMO.
In this case, I guess it is fine as it is.